Assets/Prefabs/Game Scripts/Player.cs

```diff
@@ -1,11 +1,15 @@
 using System;
 using System.Collections;
 using System.Collections.Generic;
+using RegressionGames;
 using UnityEngine;
 
 public class Player : MonoBehaviour
 {
-
+    private Vector3? targetPosition;
+    private Rigidbody rigidbody;
```

Comment:
duplicate field.  Looks like this class already had `_rigidbody`

Reply:
Fixed